Reavas

NevasPrime opened the pull request.

NevasPrime committed:
Merge pull request #7 from ValliereMagic/master

Updating

NevasPrime committed:
Changes to be committed:
	new file:   nginx/root/static/css/form.css
- Added css for addDog
- Can be used for other animals forms
	modified:   nplbam/app/__init__.py
- Added BP for newAnimal.py
	new file:   nplbam/app/newAnimal.py
- Added route
- Needs to have database loading from forum.
	new file:   nplbam/app/templates/addDog.html
- Added forms
- Need to selectors
- Need to add attach files
- Need to add attach pictures

- Need to add database column for Vet Notes

ValliereMagic commented on nplbam/app/__init__.py:
```diff
@@ -13,11 +13,13 @@ def create_app():
     # Set the secret key from the config file
     app.secret_key = config.SECRET_KEY
-    from . import main, animals
+    from . import main, animals, newAnimal
```
Is there a reason why we are using lower-camel-case here? Currently the rest of the project is using snake case, and I believe in the style of python snake case may make more sense in this instance.
https://stackoverflow.com/questions/8908760/should-i-use-camel-case-or-underscores-in-python

ValliereMagic replied:
Is there a reason why we are using lower-camel-case here? Currently the rest of the project is using snake case, and I believe in the style of python snake case may make more sense in this instance.
https://stackoverflow.com/questions/8908760/should-i-use-camel-case-or-underscores-in-python

ValliereMagic commented on nplbam/app/newAnimal.py:
```diff
@@ -0,0 +1,9 @@
+bp = Blueprint('newAnimal', __name__, url_prefix="")
+
+@bp.route("/newAnimal")
+def query():
```
Does 'query' make sense as a method name for this page? or is this a mistake from copying and pasting from another file?

ValliereMagic commented on nplbam/app/newAnimal.py:
```diff
@@ -0,0 +1,9 @@
+def query():
+    if flask_session.get("userID", default=None) is None:
+        return redirect("/")
+    return render_template("addDog.html", title="Add Dog")
```
Again camel case here

ValliereMagic commented on nplbam/app/templates/addDog.html:
```diff
@@ -0,0 +1,316 @@
+            </form>    
+        </main>
+    </body>
+</html>
```
Definitely will need more extended use of templating in the future.

NevasPrime committed:
Changes to be committed:
	modified:   nplbam/app/__init__.py
- Changed newAnimal to new_animal
- Redirect to /animals if already logged in
	modified:   nplbam/app/db/db.py
- creates database when run now
	renamed:    nplbam/app/newAnimal.py -> nplbam/app/new_animal.py
	renamed:    nplbam/app/templates/addDog.html -> nplbam/app/templates/add_dog.html
	modified:   nplbam/app/templates/animals.html
- Changed newAnimal/addDog to be named new_animal and add_dog

NevasPrime committed:
Changes to be committed:
	modified:   nplbam/app/db/db.py
- Fixed my mistake

NevasPrime committed:
modified:   nplbam/app/main.py
- Main now redirects if already logged in
	modified:   nplbam/app/new_animal.py
- Changed method name from "query"
